### PR TITLE
Add colors to output

### PR DESCRIPTION
**Pull request by ilya112358**
Please, check if these colors work in linux shell and look ok for you.

### CODE REVIEW

**Comment by fyodor7878** [comment]
I had to make python 3.11 from source since Debian has 3.9 still. But that did not help anyway since I used sqlite library from Debian repo which, again, is not updated (3.34) and does not support new commands that you use. I worked it out eventually but I suggest that you fall back with those commands since they are not widely supported yet.

As for colors, I think, it is a nice touch, but expect them to look different on different platforms. In my environment I would prefer them to be bright, the normal ones are too dark (see screenshot for comparison). 
SvenskaJa Screenshot 2023-01-16 131407
Also, don't forget to add colorama to requirements.txt.

**Comment by fyodor7878** [comment]
You asked before about Unicode under Linux:
https://www.cl.cam.ac.uk/~mgk25/unicode.html

**Comment by ilya112358** [comment]
You should be able to run it on 3.9 now. I decided to leave red and green as they are but brighten yellow.

File: practice.py [hunk 1]
```diff
@@ -218,10 +223,10 @@ def __init__(self, verbs):
 
                 def question(self, verb: str, answer: str) -> bool:
                     """Check knowledge of a translation"""
-                    print('\nVerb:', verb)
+                    print(f'\n{verb}')
                     input()
```
Comment (fyodor7878) [inline practice.py:227]: Unite it into one: input(f'\n{verb}') to remove extra empty line which now looks too empty, I think.

**Review by fyodor7878 — CHANGES_REQUESTED** [review]
One suggestion, otherwise looks ok

**Review by fyodor7878 — APPROVED** [review]
Good to go

**Pull request merged**